tutor/resources/styles/book-content/questions.less

```diff
@@ -23,7 +23,7 @@ section {
     }
 
     // make fully imported questions imitate embedded exercises
-    ol {
+    > ol:not(:first-child) {
       counter-reset: lowerAlpha;
```

Comment:
What's special about a ol that's first-child ?  Could we add a comment to indicate that so future us can remember the reason?